### PR TITLE
DSERV-191-biocypher-update

### PR DESCRIPTION
**Pull request by mingjiecn**

### CODE REVIEW

File: data/tests/test_arango_db.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def test_arangodb_generates_import_statements_for_nodes(mock_client):
         db = ArangoDB()
 
         cmds = db.generate_import_statement(
-            'header-test.csv', ['data1.csv', 'data2.csv'], 'test_collection', 'node')
+            'header-test.csv', ['data1.csv', 'data2.csv'], 'test_collection', 'node', True)
```
Comment (pedrohr) [inline data/tests/test_arango_db.py:174]: we need an extra test with False just to guarantee the `--translate` parameter won't be passed

**Review by pedrohr — CHANGES_REQUESTED** [review]
We just need a test with `has_edge_id` as False.

File: data/db/arango_db.py [hunk 2]
```diff
@@ -79,7 +70,7 @@ def generate_import_statement(self, header_path, data_filenames, collection, ele
 
             if element_type == 'edge':
                 cmd += '--create-collection-type edge --translate ":START_ID=_from" --translate ":END_ID=_to"'
-                if self.has_edge_key(header_path):
+                if has_edge_id:
                     cmd += ' --translate "id=_key"'
```
[the remark this answers is not in the document]
Comment (mingjiecn) [reply data/db/arango_db.py:74]: Yes in Biocypher the default setting is True.

File: data/tests/test_arango_db.py (hunk 1 above)
Comment (pedrohr) [reply data/tests/test_arango_db.py:174]: thank you 🙏

**Review by pedrohr — APPROVED** [review]
👍

**Pull request merged**